ASP/DeliveryServiceWebApi/DeliveryServiceWebApi/Controllers/FoodController.cs

```diff
@@ -14,53 +14,53 @@ namespace DeliveryServiceWebApi.Controllers
     public class FoodController : ControllerBase
     {
         private readonly ILogger<FoodController> _logger;
-        private readonly IFoodController _controller;
+        private readonly IFoodService _service;
 
-        public FoodController(ILogger<FoodController> logger, IFoodController foodController)
+        public FoodController(ILogger<FoodController> logger, IFoodService foodService)
         {
             _logger = logger;
-            _controller = foodController;
+            _service = foodService;
         }
 
         [HttpGet]
         public IEnumerable<Food> Get()
         {
-            return _controller.Get();
+            return _service.Get();
         }
 
         [HttpGet("{id}")]
         public Food Get(int id)
         {
-            return _controller.SearchFood(id);
+            return _service.SearchFood(id);
         }
 
         [HttpPost]
         public ActionResult<Food> Post(Food food)
```

Comment:
public ActionResult<Food> Post(Food food)
->
public ActionResult Post(Food food)
or 
public IActionResult Post(Food food)

Reply:
the same withother actions